modules/virt-creating-linux-bridge-nad-cli.adoc

```diff
@@ -30,25 +30,25 @@ kind: NetworkAttachmentDefinition
 metadata:
   name: bridge-network # <1>
   annotations:
-    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/bridge-interface # <2>
+    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br1 # <2>
 spec:
   config: |
     {
       "cniVersion": "0.3.1",
       "name": "bridge-network", <3>
       "type": "bridge", <4>
-      "bridge": "bridge-interface", <5>
+      "bridge": "br1", <5>
       "macspoofchk": false, <6>
       "vlan": 100, <7>
       "disableContainerInterface": true,
       "preserveDefaultVlan": false <8>
     }
 ----
 <1> The name for the `NetworkAttachmentDefinition` object.
-<2> Optional: Annotation key-value pair for node selection, where `bridge-interface` must match the name of a bridge configured on some nodes. If you add this annotation to your network attachment definition, your virtual machine instances will only run on the nodes that have the `bridge-interface` bridge connected.
+<2> Optional: Annotation key-value pair for node selection for the bridge configured on some nodes. If you add this annotation to your network attachment definition, your virtual machine instances will only run on the nodes that have the defined bridge connected.
 <3> The name for the configuration. It is recommended to match the configuration name to the `name` value of the network attachment definition.
 <4> The actual name of the Container Network Interface (CNI) plugin that provides the network for this network attachment definition. Do not change this field unless you want to use a different CNI.
-<5> The name of the Linux bridge configured on the node.
+<5> The name of the Linux bridge configured on the node. The default Linux `NodeNetworkConfigurationPolicy` (NNCP) bridge name is `br1`.
```

Comment:
There is no default name for a linux bridge. It just happens to be the name we used in the example here https://docs.openshift.com/container-platform/4.16/virt/vm_networking/virt-connecting-vm-to-linux-bridge.html.